lib/reverse_sentence.rb

```diff
@@ -1,6 +1,55 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2)
+# Space complexity: O(1)
```

Comment:
The time complexity is O(n) since you make two passes through the string, one to reverse the entire string and then **after** that you go through the string reversing each word O(n + n) = O(n).